examples/barrett/src/lib.rs

```diff
@@ -38,7 +38,7 @@ pub fn barrett_reduce(value: FieldElement) -> FieldElement {
     // assert!(((quotient as i64) * (FIELD_MODULUS as i64)) < 9223372036854775807);
     let sub = quotient * FIELD_MODULUS;
 
-    hax::fstar!(r"Math.Lemmas.cancel_mul_mod (v $quotient) 3329");
+    hax::fstar_expr!(r"Math.Lemmas.cancel_mul_mod (v $quotient) 3329");
```

Comment:
Why was this renamed? `fstar` looks nicer.

Reply:
That was to disambiguate between e.g. `fstar` and `fstar_before`, but yeah, that makes no much sense after all, I can revert to just `fstar`!